build.gradle

```diff
@@ -67,7 +67,7 @@ repositories {
 dependencies {
 	// Terra deps
 	implementation group: "bio.terra", name: "datarepo-client", version: "1.0.1-SNAPSHOT"
-	implementation group: "bio.terra", name: "stairway", version: "0.0.25-SNAPSHOT"
+	implementation group: "bio.terra", name: "stairway", version: "0.0.35-SNAPSHOT"
```

Comment:
worth double-checking since this is a 10 release jump: are there any major/breaking changes we should know about, or has stairway been pretty stable?

Reply:
The numbers bump pretty fast, since every merge causes a version bump. This is all upward compatible: addition of debuginfo, retry logging, and the hook enhancements.